.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: olafurpg/setup-scala@v5
+
+    - name: Cache Coursier dependencies
+      uses: actions/cache@v1
+      with:
+       path: ~/.cache/coursier
+       key: ${{ runner.os }}-${{ github.ref }}
```

Comment:
I found this name collides with the other cache, they need different names. See https://github.com/profunktor/http4s-jwt-auth/commit/b4965c94bd9a32f7e474ea98a1899df9e3d61d08

Reply:
Updated accordingly, however with Sbt 1.3.x the .ivy2 resolution now uses coursier behind the scenes as the dependency management repo by default, so I think we only need the .ivy2 to be cached unless I am missing something.